tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -171,6 +171,13 @@ public:
 
       RValue GetNonOwningCopy() { return RValue(fField, fObjPtr, false); }
 
+      template <typename T>
+      void *Release()
+      {
+         fIsOwning = false;
+         return static_cast<T *>(fObjPtr);
```

Comment:
Should this also set `fObjPtr` to `nullptr`?
The semantic of the current code is less a 'releasing' of the data and more of a 'switch to shared use'.
In particular if the user than read the next entry, the value will still change.  This may or may not be the intended behavior.  There might be the need for both  (`ReleaseJustOwnership` and `ReleaseAndForget`)

Reply:
From my side there are no objections to setting the pointer to null. Right now the RValue object goes out of scope immediately after I take the pointer.

Reply:
Done